documentation/CLI-for-JFrog-Curation.md

```diff
@@ -0,0 +1,81 @@
+JFrog CLI : CLI for JFrog Curation
+======================================
+
+
+Overview
+--------
+
+This page describes how to use JFrog CLI with JFrog Curation.
+
+Read more about JFrog CLI [here](https://jfrog.com/help/r/jfrog-cli).
+
+---
+**Note**
+> JFrog Curation is only available since [Artifactory 7.63.2](https://jfrog.com/help/r/jfrog-release-information/artifactory-7.63.2-cloud) And [Xray 3.78.9](https://jfrog.com/help/r/jfrog-release-information/xray-3.78.9).
+
+---
+
+### Syntax
+
+When used with JFrog Distribution, JFrog CLI uses the following syntax:
+
+	$ jf ca command-name command-options 
+
+Where:
+
+
+|                 |                                                                                                 |
+|-----------------|-------------------------------------------------------------------------------------------------|
+| command-name    | The command to execute. Note that you can use either the full command name or its abbreviation. |
+| command-options | A set of options corresponding to the command                                                   |
+
+
+
+### Commands
+
+The following sections describe the commands available in the JFrog CLI for use with JFrog Curation.
+
+Curation-Audit
+---------------------
+**Note**
+>The command _curation-audit_ currently supports only [npm](https://www.npmjs.com/) projects.
```

Comment:
The command _curation-audit_ currently...
-->
The **curation-audit** command  currently...